src/main/frontend/handler/repo.cljs

```diff
@@ -506,14 +484,8 @@
 
 (defn periodically-push-tasks
   [repo-url]
-  (spec/validate :repos/url repo-url)
-  (let [push (fn []
-               (when (and (not (false? (:git-auto-push (state/get-config repo-url))))
-                          ;; (not config/dev?)
-)
-                 (push repo-url nil)))]
-    (js/setInterval push
-                    (* (config/git-push-secs) 1000))))
+  (js/setInterval #(push-if-auto-enabled! repo-url)
```

Comment:
The `(spec/validate :repos/url repo-url)` was removed.

Reply:
`push-if-auto-enabled!` has the spec. Do we need another one here?

Reply:
ah my bad, ignore it.